README.md

```diff
@@ -0,0 +1,84 @@
+# bdk-dart
+
+Dart bindings for the [Bitcoin Dev Kit (BDK)](https://bitcoindevkit.org/) wallet library.
+The repo packages the generated UniFFI bindings (`lib/bdk.dart`) together with the
+compiled `libbdkffi` native library so Dart and Flutter apps can work with descriptor-based
+wallets, key management utilities, and blockchain backends from BDK.
+
+## Repository layout
+
+| Path | Purpose |
+| ---- | ------- |
+| `bdk-ffi/` | Rust sources and build scripts for the underlying `bdk-ffi` crate. |
+| `lib/` | Generated Dart bindings (`bdk.dart`) produced by UniFFI-Dart. |
+| `examples/` | Standalone Dart examples that exercise common workflows. |
+| `test/` | Integration-style tests that cover wallet creation, persistence, and networking. |
```

Comment:
maybe we change this to something like `Dart unit tests for wallet construction, offline behavior, persistence` or something since we only have offline tests at the moment?